AUTHORS.rst

```diff
@@ -41,18 +40,17 @@ Many thanks for your contributions!
 There are also a few modules or functions incorporated from other
 authors and projects:
 
-* ``utils.iso_precision`` includes something like 10 lines of Arrow's, code,
-  which is written by Chris Smith and under Apache license
+* The function ``ics.utils.iso_precision`` includes something like 10 lines of
```

Comment:
We can probably drop this paragraph entirely, as the `iso_precision` function is gone together with arrow (we now get the precision directly from reading the `VALUE` type parameter).

Reply:
Yes!

Reply:
```suggestion
```